config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
```

Comment:
again, this works, so yay :-)  you may want to use [versionist](https://github.com/bploetz/versionist) like we do on [highlights](https://github.com/openstax/highlights-api/blob/master/Gemfile#L14) and search

Reply:
added versionist - still trying to figure out what makes this better :D